File: generator/output/ts-model/pay-order.model.ts
class PayOrderModel {
  public id: number = 0;

  // snapshot 快照
  public snapshot?: string;

  // deleted 是否删除
  public deleted?: number = 0;

  // trade_no 订单号
  public trade_no: string;

  // price 价格（单位元）
  public price: number = 0;

  // subject_id subject id
  public subject_id: string;

  // pay_time 支付时间
  public pay_time?: string;

  // type 购买类型
  public type: string;

  // is_pay 是否已经支付
  public is_pay: boolean = false;

  // user_id 用户id
  public user_id: number = 0;

  public created_at: string;

  public updated_at: string;
}

const PayOrderModelFields = {
  id: "id",
  snapshot: "snapshot",
  deleted: "deleted",
  trade_no: "trade_no",
  price: "price",
  subject_id: "subject_id",
  pay_time: "pay_time",
  type: "type",
  is_pay: "is_pay",
  user_id: "user_id",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {PayOrderModel, PayOrderModelFields};

File: backend/src/main/java/com/licenseair/backend/domainModel/PayOrderModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Long;
import java.math.BigDecimal;
import java.lang.Boolean;
import java.sql.Timestamp;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* PayOrder
*/
@Data()
public class PayOrderModel {
  public Long id = null;

  /**
  * 快照
  */
  public String snapshot = null;

  /**
  * 是否删除
  */
  public boolean deleted = false;

  /**
  * 订单号
  */
  public String trade_no = null;

  /**
  * 价格（单位元）
  */
  public BigDecimal price = null;

  /**
  * subject id
  */
  public Long subject_id = null;

  /**
  * 支付时间
  */
  public Timestamp pay_time = null;

  /**
  * 是否已经支付
  */
  public Boolean is_pay = null;

  /**
  * 用户id
  */
  public Long user_id = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}
